llvm/lib/CodeGen/SelectionDAG/SelectionDAG.cpp

```diff
@@ -13267,7 +13267,9 @@ SDValue SelectionDAG::getNeutralElement(unsigned Opcode, const SDLoc &DL,
   case ISD::SMIN:
     return getConstant(APInt::getSignedMaxValue(VT.getSizeInBits()), DL, VT);
   case ISD::FADD:
-    return getConstantFP(-0.0, DL, VT);
+    // If flags allow, prefer positive zero single it's generally cheaper
```

Comment:
Weird typo "single" for "since"?

Reply:
Fixed in e1bde1c5b203224b0fa68ee141ec2c7a079f5eac